make/autoconf/otp.m4

```diff
@@ -106,6 +106,7 @@ AC_ARG_VAR(DED_LD_FLAG_RUNTIME_LIBRARY_PATH, [runtime library path linker flag f
 AC_ARG_VAR(RANLIB, [ranlib])
 AC_ARG_VAR(AR, [ar])
 AC_ARG_VAR(GETCONF, [getconf])
+AC_ARG_VAR(EX_DOC, [Path to ex_doc executable])
```

Comment:
This is the only line I changed. Everything else is autogenerated by `./otp_build update_configure`